File: java/src/com/vittee/thainum/provider/DefaultDigitNameProvider.java
package com.vittee.thainum.provider;

public class DefaultDigitNameProvider extends BaseNameProvider implements DigitNameProvider {

    public DefaultDigitNameProvider() {
        super("ศูนย์", "หนึ่ง", "สอง", "สาม", "สี่", "ห้า", "หก", "เจ็ด", "แปด", "เก้า");
    }
}

File: csharp/ThaiNum/ThaiNum.Library/Provider/DefaultDigitNameProvider.cs
namespace ThaiNum.Library.Provider
{
    public class DefaultDigitNameProvider : BaseNameProvider, IDigitNameProvider
    {
        public DefaultDigitNameProvider() 
            : base("ศูนย์", "หนึ่ง", "สอง", "สาม", "สี่", "ห้า", "หก", "เจ็ด", "แปด", "เก้า")
        {
            
        }
    }
}
